gcloud-java-dns/src/main/java/com/google/gcloud/dns/DnsOptions.java

```diff
@@ -37,7 +37,7 @@
     @Override
     public Dns create(DnsOptions options) {
       // TODO(mderka) Implement when DnsImpl is available. Created issue #595.
-      return null;
+      return new DnsImpl(options);
```

Comment:
The comment for sure can go. However, I want to verify that this class is complete before closing the issue itself.